Can't run config --init

`sudo snap install clockify-cli`

When I run the init command and pass my TOKEN I get the correct workspace but when I press return I get the following error.

```
➜  ~ clockify-cli config --init
? User Generated Token: MY_TOKEN
? Choose default Workspace:
panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
github.com/lucassabreu/clockify-cli/cmd.configInit(0xc2e320, 0xc000162830, 0x0, 0x1, 0x0, 0x0)
	/home/runner/work/clockify-cli/clockify-cli/cmd/config.go:172 +0xe1e
github.com/lucassabreu/clockify-cli/cmd.glob..func3(0xc2e320, 0xc000162830, 0x0, 0x1, 0x0, 0x0)
	/home/runner/work/clockify-cli/clockify-cli/cmd/config.go:69 +0x11e
github.com/spf13/cobra.(*Command).execute(0xc2e320, 0xc000162820, 0x1, 0x1, 0xc2e320, 0xc000162820)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc2d6a0, 0x0, 0xc000036060, 0x70)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/lucassabreu/clockify-cli/cmd.Execute(0x99be64, 0x6, 0x9ac420, 0x28, 0x99db40, 0x14)
	/home/runner/work/clockify-cli/clockify-cli/cmd/root.go:51 +0xb2
main.main()
	/home/runner/work/clockify-cli/clockify-cli/main.go:26 +0x69
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- select UI component can fail to return a valid option if the default value were not in the list, to prevent
+  that if the default value is empty or not in the list, no default value will be set.
+
 ## [v0.19.4] - 2021-07-21
 
 ### Fixed

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v0.19.5] - 2021-08-03
+
 ### Fixed
 
 - select UI component can fail to return a valid option if the default value were not in the list, to prevent
@@ -555,7 +557,9 @@ time entry.
 - Golang CLI using [cobra](https://github.com/spf13/cobra)
 - Makefile to help setup actions
 
-[Unreleased]: https://github.com/lucassabreu/clockify-cli/compare/v0.19.3...HEAD
+[Unreleased]: https://github.com/lucassabreu/clockify-cli/compare/v0.19.5...HEAD
+[v0.19.5]: https://github.com/lucassabreu/clockify-cli/releases/tag/v0.19.5
+[v0.19.4]: https://github.com/lucassabreu/clockify-cli/releases/tag/v0.19.4
 [v0.19.3]: https://github.com/lucassabreu/clockify-cli/releases/tag/v0.19.3
 [v0.19.2]: https://github.com/lucassabreu/clockify-cli/releases/tag/v0.19.2
 [v0.19.1]: https://github.com/lucassabreu/clockify-cli/releases/tag/v0.19.1

--- a/ui/ui.go
+++ b/ui/ui.go
@@ -33,12 +33,17 @@ func AskForText(message, d string) (string, error) {
 
 // AskFromOptions interactively ask the user to choose one option or none
 func AskFromOptions(message string, options []string, d string) (string, error) {
-	return askString(&survey.Select{
+	p := &survey.Select{
 		Message: message,
 		Options: options,
-		Default: d,
 		Filter:  selectFilter,
-	})
+	}
+
+	if d != "" && strhlp.Search(d, options) != -1 {
+		p.Default = d
+	}
+
+	return askString(p)
 }
 
 // AskManyFromOptions interactively ask the user to choose none or many option

